ui/src/pages/namespace/Explorer/Tree/index.tsx

```diff
@@ -44,7 +45,23 @@ const ExplorerPage: FC = () => {
   const [deleteNode, setDeleteNode] = useState<BaseFileSchemaType>();
   const [renameFile, setRenameNode] = useState<BaseFileSchemaType>();
   const [previewNode, setPreviewNode] = useState<BaseFileSchemaType>();
+
+  const [search, setSearch] = useState("");
```

Comment:
```suggestion
  const [query, setQuery] = useState("");
```
I find this a bit clearer. I did not comment on repeated uses of these names.